### ISSUE TITLE
Fake Telegram Copyright Reports

### ISSUE THREAD

**Issue by anasty17** [telegramdesktop/tdesktop#26269]
### Steps to reproduce

Produce some haters

### Expected behaviour

Ping telegram support employees or volunteers
Maybe some coffee or hit them on the brain 

### Actual behaviour

I'm owner of a github repository which is http://github.com/anasty17/mirror-leech-telegram-bot 

I have telegram channel linked to group. Channel for commits update and group for support.

Someone made fake copyright reports and i have reported to tdlib repository on telegram because NO ONE RESPONDING TO MY EMAILS.

Here all evidences:

https://github.com/tdlib/td/issues/2420

### Operating system

Same as telegram support, windows 95

### Version of Telegram Desktop

Support version is under 0

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
this is telegram desktop bug tracker, not "client" support place.

**Comment by anasty17** [comment]
> this is telegram desktop bug tracker, not user support place.

I know this. But there is no support place in telegram

**Comment by anasty17** [comment]
I have repository with 5.4k fork i know how much this confusing the dev. But also no one responding from telegram support on those fake coptright strikes so I'm talking here trying to reach someone by ur help.

I have tried on twitter for login issues but they gave automated message lol.
I have tried on email.
I have tried on ask question from telegram app.

And no response, now I'm looking to wait new twitter post to reply because the account holder only reply to first 4 or 5 replies.
